### PR TITLE
Streetview panorama options

### PR DESCRIPTION
**Pull request by offtherailz**
## Description
This PR adds the possibility to customize the panorama options. This have to be configured in every plugin's configuration. 

**Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines: https://github.com/geosolutions-it/MapStore2/blob/master/CONTRIBUTING.md
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x", remove the others)
 - [ ] Bugfix
 - [x] Feature
 - [ ] Code style update (formatting, local variables)
 - [ ] Refactoring (no functional changes, no api changes)
 - [ ] Build related changes
 - [ ] CI related changes
 - [ ] Other... Please describe:

## Issue

**What is the current behavior?**

#<issue>

**What is the new behavior?**

## Breaking change
**Does this PR introduce a breaking change?** (check one with "x", remove the other)
 - [ ] Yes, and I documented them in migration notes
 - [x] No

### CODE REVIEW

File: web/client/plugins/StreetView/selectors/streetView.js [hunk 1]
```diff
@@ -40,10 +40,18 @@ export function googleAPIKeySelector(state) {
...
 export function useStreetViewDataLayerSelector(state) {
     return streetViewConfigurationSelector(state)?.useDataLayer ?? true;
 }
 
+/**
+ * gets from the state the panorama options currently loaded
+ * @param {object} state
+ * @returns the panorama options configured
+ */
+export function panoramaOptionsSelector(state) {
```
Comment (MV88) [inline web/client/plugins/StreetView/selectors/streetView.js:55]: you want to add a unit test here?

**Review by MV88 — CHANGES_REQUESTED** [review]
@offtherailz 
maybe you can add a unit test
otherwise feel free to merge

**Review by MV88 — APPROVED** [review]

**Comment by MV88** [comment]
@ElenaGallo please test in DEV

**Comment by MV88** [comment]
@offtherailz which is the associated issue?

**Comment by offtherailz** [comment]
Created and associated. thanks

**Comment by ElenaGallo** [comment]
Test passed, @offtherailz please backport to 2021.02.xx. Thanks

**Pull request merged**